workflow/rules/common.smk

```diff
@@ -220,11 +220,22 @@ def get_target_bed_statement(wildcards):
 
 
 def get_target_regions(wildcards):
-    benchmark = get_benchmark(wildcards.benchmark)
-    if "target-regions" in benchmark:
-        return "resources/regions/{benchmark}/target-regions.bed"
+    if hasattr(wildcards, "benchmark"):
+        benchmark = get_benchmark(wildcards.benchmark)
+        if "target-regions" in benchmark:
+            return "resources/regions/{benchmark}/target-regions.bed"
+        else:
+            return []
     else:
-        return []
+        benchmark_name = config["variant-calls"][wildcards.callset]["benchmark"]
+        benchmark = get_benchmark(benchmark_name)
+        if "target-regions" in benchmark:
+            # TODO use f-string when this is fixed: https://github.com/snakemake/snakefmt/issues/215
```

Comment:
Isn't this fixed now? The issue is closed.

Reply:
I removed all TODOs and added `f`